Sources/WebSockets/Connection/WebSockets+Hash.swift

```diff
@@ -1,5 +1,5 @@
-import SHA1
-import CryptoEssentials
+import CLibreSSL
```

Comment:
Maybe worth it to bring in Crypto here. It's only a few Swift files and reduces possible error prone code.